packages/sol/src/query.ts

```diff
@@ -0,0 +1,18 @@
+const lessonIdKey = "rate-lesson-id";
+const tutorIdKey = "rate-tutor-id";
+
+export function asRateLessonQuery({
+  lessonId,
+  tutorId,
+}: {
+  lessonId: number;
+  tutorId: number;
+}) {
+  return `?${lessonIdKey}=${lessonId}&${tutorIdKey}=${tutorId}`;
```

Comment:
The `?` should be provided by the consumer. 

```suggestion
  return `${lessonIdKey}=${lessonId}&${tutorIdKey}=${tutorId}`;
```